--- CMakeLists.txt	2022-02-10 20:55:35.503517231 +0100
+++ CMakeLists.txt.~1~	2022-02-10 20:55:23.273461350 +0100
@@ -172,9 +172,9 @@
 file(GLOB NCCORESRCS CONFIGURE_DEPENDS src/lib/*.c src/lib/*.cpp)
 add_library(notcurses-core SHARED ${NCCORESRCS} ${COMPATSRC})
 if(${USE_STATIC})
-add_library(notcurses-core-static STATIC ${NCCORESRCS})
+add_library(notcurses-core-static STATIC ${NCCORESRCS} ${COMPATSRC})
 else()
-add_library(notcurses-core-static STATIC EXCLUDE_FROM_ALL ${NCCORESRCS})
+add_library(notcurses-core-static STATIC EXCLUDE_FROM_ALL ${NCCORESRCS} ${COMPATSRC})
 endif()
 # don't want these on freebsd/dragonfly/osx
 if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
@@ -261,9 +261,9 @@
 file(GLOB NCSRCS CONFIGURE_DEPENDS src/media/*.c src/media/*.cpp)
 add_library(notcurses SHARED ${NCSRCS} ${COMPATSRC})
 if(${USE_STATIC})
-add_library(notcurses-static STATIC ${NCSRCS})
+add_library(notcurses-static STATIC ${NCSRCS} ${COMPATSRC})
 else()
-add_library(notcurses-static STATIC EXCLUDE_FROM_ALL ${NCSRCS})
+add_library(notcurses-static STATIC EXCLUDE_FROM_ALL ${NCSRCS} ${COMPATSRC})
 endif()
 set_target_properties(notcurses PROPERTIES
   VERSION ${PROJECT_VERSION}
